README.md

```diff
@@ -1 +1,53 @@
-# oop-school-liberary
\ No newline at end of file
+# enumerable
+
+## Welcome! 👋
+                              
+## Built With ✔️
+
+- Ruby
+
+## Functionalites 😄
+
+- You can add different codes and check them.
+
+
+## Getting Started 🙌
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+```
+  Ruby
+
+```
+### Setup
+Clone the repository and change the directory to portfolio
+
+``` 
+  `$ git clone 'https://github.com/safar1212/oop-school-liberary'`
+
+  `$ cd oop-school-liberary`
+
+  `$ irb`
+
+```
+
+
+### Authors
+
+| 👤 Name | Github | Twitter | LinkedIn |
+|------|--------|---------|----------|
+|Safar Ali|[@Safar Ali](https://github.com/safar1212)|[@Safar Ali](https://twitter.com/SafarAli999)|[@Safar Ali](https://www.linkedin.com/in/safar-ali999/)|
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is owned by Safar Ali.
+
```

Comment:
Hey, since the last reviewer didn't raise this as a requirement I am marking this as optional. However, I strongly recommend you implement this.
- Kindly update your README file as per this project, like title, description, prerequisites, setup, and any other section that is relevant to this project. Remember it is super important to have a good README file.